db/pf-schema-X.Y.Z.sql

```diff
@@ -1107,6 +1107,78 @@ CREATE TABLE radius_audit_log (
 ) ENGINE=InnoDB;
 
 --
+-- Table structure for table `node_option82`
+--
+
+CREATE TABLE `node_option82` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
+  `created_at` TIMESTAMP NOT NULL,
+  `mac` varchar(17) NOT NULL,
+  `option82_switch` varchar(17) NULL,
+  `switch_id` varchar(17) NULL,
+  `port` varchar(8) NOT NULL default '',
+  `vlan` varchar(255) default NULL,
+  `circuit_id_string` varchar(255) default NULL,
+  `module` varchar(255) default NULL,
+  `host` varchar(255) default NULL,
+  UNIQUE KEY mac (mac)
+) ENGINE=InnoDB;
+
+--
+-- Table structure for table `node_option82_history`
+--
+
+CREATE TABLE `node_option82_history` (
+  `node_option82_id` int NOT NULL PRIMARY KEY AUTO_INCREMENT,
```

Comment:
Yes it would create multiple entries in the history table when an option 82 entry has changed.
That is by design to keep track of all the changes for a mac.

To answer your question directly keeping track of the node_option82_id does not give us anything.

We could either drop node_option82_id all together and just have the mac as the primary key.